olp-cpp-sdk-authentication/src/SignInResultImpl.cpp

```diff
@@ -91,5 +103,63 @@ const std::string& SignInResultImpl::GetScope() const { return scope_; }
 
 bool SignInResultImpl::IsValid() const { return is_valid_; }
 
+std::string SignInResultImpl::ParseJwtToken(const std::string& token) {
+  std::string result;
```

Comment:
```suggestion
```
No need to initialize this here - better create this var when it is actually used - at line 136